CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+## [5.0.1] - 2018-11-07
+
+### Changed
+
+* Removing support for old API keys
+
+## [5.0.0] - 2018-11-01
+
+### Changed
+
+* Re-purposing for Notify.gov.au
+* Removed support for letters
+* Converted to typescript
+
 ## [4.5.1] - 2018-09-14
```

Comment:
I think we need to make it clear somehow that these releases don't actually exist under our NPM package. Perhaps we just remove them from the CHANGELOG altogether—they are not actually changes as far as our brand new NPM package is concerned. Maybe we just put a markdown link to the GDS CHANGELOG#4.5.1 fragment?